dataframes/04-reading-messy-data-into-dataframes.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
<p>Perhaps add a conclusion paragraph recapping what was done (using <code>dask.delayed</code>  and <code>dask.dataframe.from_delayed</code>  to create a custom reader function that handles messy data). The end here feels a bit abrupt.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dask/dask-examples/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>good idea</p>